uiowa/uiowa
# OnIowa - events grouped by date

Resolves #6607 
Resolves #6646 

# How to test
```
ddev blt ds --site=oniowa.uiowa.edu && ddev drush @oniowa.local uli events
```
- See events are grouped together under date headings.
- Check that the time only - no date - is displaying for each individual event under a date. Uunless it is a multiday event, in which case the entire start date and end date should display instead.
- Click on an event and make sure the entire date and time still displays on its page. Add a presenter to an event and check the entire date and time still displays on their page too.
- Date format should now resemble Monday, July 17, 2023 for the date headings and the teasers - including on an individual event's page and a presenter's page.

## Date option/form caching
- Set a breakpoint at line 22 in `oniowa_core.module`.
- Turn on "Start Listening for PHP Debug Connections"
- Change line 42 to:
```php
$expire = strtotime('now') + 60;
```
```
ddev drush @oniowa.local cr && ddev xdebug
```
- https://oniowa.uiowa.ddev.site/events
- Note that breakpoint on line 22 is triggered.
- Set a timer for 60 seconds.
- Immediately refresh the page. Breakpoint should not be triggered.
- After the timer goes off, refresh the page again. Breakpoint should be triggered again.
